policy/cnspec_policy.proto

```diff
@@ -518,6 +518,7 @@ service PolicyHub {
   rpc SetBundle(Bundle) returns (Empty) {}
   rpc ValidateBundle(Bundle) returns (Empty) {}
   rpc GetBundle(Mrn) returns (Bundle) {}
+  rpc GetBundleForMrn(BundleReq) returns (Bundle) {}
```

Comment:
nit: can we call this `GetBundleV2`? no real preference here, just sounds better to me 🤷